src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -471,8 +471,16 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 				// The connected succeeded so add it to our active connections now, optionally adding it to the MRU based on
 				// the options.saveTheConnection setting
 				let connectionMgmtInfo = this._connectionStatusManager.findConnection(uri);
+				if (!connectionMgmtInfo) {
+					this._logService.info(`Could not find connection management info for ${uri} after connection`);
+					return connectionResult;
```

Comment:
Just a quick question, will this early return impact the following code paths for saving profile/password? Or are those code paths not reachable when `connectionMgmtInfo` is not available?

Reply:
Oops, meant to take that out. Was left over from testing the fix. Thanks for catching!